src/diffpy/srmise/modelcluster.py

```diff
@@ -151,8 +153,10 @@ def transform(self, in_format, out_format, **kwds):
 
         Parameters
         ----------
-        in_format - The current format of parameters
-        out_format - The new format for parameters
+        in_format : Any
+            The current format of parameters
```

Comment:
Are these in put and output parameters?  If you can figure out what the parameters are please make this more informative.